addons/quickmount/functions/fnc_getInNearest.sqf

```diff
@@ -0,0 +1,50 @@
+/*
+ * Author: Kingsley
+ * Mount the given player in the nearest eligible vehicle based on their distance and direction to the vehicle
+ *
+ * Arguments:
+ * 0: Player unit <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [ace_player] call acex_quickmount_fnc_getInNearest;
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params [["_unit", objNull, [objNull]]];
+
+if (!GVAR(Enabled)) exitWith {};
+if (isNull _unit || {vehicle _unit != _unit} || {!alive _unit} || {!isPlayer _unit}) exitWith {};
+
+private _start = eyePos _unit;
```

Comment:
This position coupled with `getCameraViewDirection` will give unexpected results on third person view.

Instead use:

```sqf
private _start = AGLtoASL positionCameraToWorld [0,0,0];
```